From 3dd33a114272fb6d3945157cccc0cc405ee570f8 Mon Sep 17 00:00:00 2001
From: Zhang Rui <rui.zhang@intel.com>
Date: Thu, 14 Feb 2019 21:35:06 +0800
Subject: [PATCH 009/108] intel_idle: add support for Jacobsville

commit 04b1d5d098491244f506c4265cc95b87210eef2f upstream.

Jacobsville uses the same C-states as Denverton.

Signed-off-by: Zhang Rui <rui.zhang@intel.com>
Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
[Yongxin Liu: use old macro instead of new one to backport to 4.18.45]
Integrated-by: Yongxin Liu <yongxin.liu@windriver.com>
---
 drivers/idle/intel_idle.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/idle/intel_idle.c b/drivers/idle/intel_idle.c
index c4bb67ed8da3..d2f154f12f31 100644
--- a/drivers/idle/intel_idle.c
+++ b/drivers/idle/intel_idle.c
@@ -1106,6 +1106,7 @@ static const struct x86_cpu_id intel_idle_ids[] __initconst = {
 	ICPU(INTEL_FAM6_ATOM_GOLDMONT,		idle_cpu_bxt),
 	ICPU(INTEL_FAM6_ATOM_GOLDMONT_PLUS,	idle_cpu_bxt),
 	ICPU(INTEL_FAM6_ATOM_GOLDMONT_X,	idle_cpu_dnv),
+	ICPU(INTEL_FAM6_ATOM_TREMONT_X,		idle_cpu_dnv),
 	{}
 };
 
-- 
2.17.1

